Makefile

```diff
@@ -9,11 +9,11 @@ all: clean test package
 clean:
 	mvn clean
 
-verify: clean
-	mvn -U verify
+install: clean
+	mvn -U install
 
 docker: clean
-	mvn install dockerfile:build dockerfile:push -DskipTests -DskipITs -Dtag=${tag} \
```

Comment:
The call to install here was not working as we are specifying the --file arg as the rs-server/pom.xml

This means that the other modules within this project are not being rebuilt, and therefore we do not get the latest jars of the validation module etc.